db/backup.go

```diff
@@ -23,10 +23,12 @@ const (
 
 var backupRegex = regexp.MustCompile(backupRegexString)
 
+const backupSuffixLayout = "2006.01.02_15.04.05"
```

Comment:
Be nice for this to be configurable?

Reply:
What is the benefits of making this configurable? Remember, it would be one more in the sea of options we already have, and this PR is already adding 3 new ones.

I'm basing this on the *arr suite, and IMO it seems a sensible value.

Reply:
It seems fine, I would personally switch it out to full ISO8601 so I can just add the backup target folder into my backup script and it'd work without any additional messing then. (It basically deals with xxxxx_ISO8601 files). But It's not essential if it seems unnecessary

Reply:
The issue with ISO8601 and RFC3339 is that they contain `:`, which may cause issues for some Windows users, if they don't know how to handle these correctly.

Reply:
Yeah I agree, I'm not suggesting the default be changed due to the reasons you've mentioned. Like I say, it's not critical at all, if you don't think it's worth it then don't bother.